apache/zeppelin
# [ZEPPELIN-606] Add a button to explicitly save a notebook

### What is this PR for?
This PR adds a button to explicitly save a notebook (without waiting for the autosave timer)
    
### What type of PR is it?
Improvement
    
### Todos
- Add a button to explicitly save a notebook (without waiting for the autosave timer)
    
### What is the Jira issue?
https://issues.apache.org/jira/browse/ZEPPELIN-606
    
### How should this be tested?
- Start Zeppelin
- Open a notebook in two tabs (or browsers)
- Make changes to a notebook on one tab and use the save button
- Verify that the changes are visible in the other tab (or browser)

### Screenshots
<img width="1372" alt="zeppelin-906-screenshot" src="https://cloud.githubusercontent.com/assets/815664/17155795/ea33c7a8-5386-11e6-98a9-974c0e41f535.png">


### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No
